package.json

```
@@ -30,10 +31,12 @@
   "dependencies": {
     "@modelcontextprotocol/sdk": "~1.5",
     "axios": "~1.8",
+    "dart-tools": "~0.3.1",
```

No need to include patch version when approx
```suggestion
    "dart-tools": "~0.3",
```